src/test/java/karuna/karuna_backend/visitor/message/domain/VisitorMessageServiceTest.java

```diff
@@ -24,4 +31,20 @@ void shouldSendMessage() {
         assertEquals(Constants.BODY, visitorMessageDto.body());
         assertEquals(Constants.CONTACT, visitorMessageDto.contact());
     }
+
+
+    @Test
+    void shouldReceiveShortenMessage() {
+        //given
+        VisitorMessageCreateDto visitorMessageCreateDto = new VisitorMessageCreateDto(Constants.LONG_BODY, Constants.CONTACT);
+        VisitorMessageDto visitorMessageDto = visitorMessageService.sendMessage(visitorMessageCreateDto);
+        VisitorMessageRequest visitorMessageRequest = new VisitorMessageRequest(PageRequest.of(0, 10, Sort.Direction.DESC, "createdAt"), 5);
+        //when
+        VisitorMessageWrapper shortenedMessages = visitorMessageService.getMessages(visitorMessageRequest);
+        List<VisitorMessageDto> messages = shortenedMessages.messages();
+        //then
+        for (VisitorMessageDto message : messages) {
+            assertEquals(5, message.body().length());
```

Comment:
Last cosmetic think, add before this loop assertion which check size of messages is greater than 1, becuase this loop can be not invoked and test give false success result

Reply:
corrected :)